index.js

```diff
@@ -11,6 +11,9 @@ const app = new App({
   signingSecret: process.env.SLACK_SIGNING_SECRET
 });
 
+const blocks = require('./blocks.js');
+const honeybadger = require('honeybadger');
```

Comment:
Looks like this line got added in from the PR. The HoneyBadger import still exists, so we should probably take it out.

```suggestion
```